.github/workflows/ci.yml

```
@@ -38,10 +38,10 @@ jobs:
       with:
         maven-version: 3.9.6
    
-    - name: Set up JDK 17
+    - name: Set up JDK 21
       uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)